src/main/scala/ru/gaptrakhmanov/telegram/helper/commands/notes/repository/DoobieUserRepository.scala

```diff
@@ -0,0 +1,68 @@
+package ru.gaptrakhmanov.telegram.helper.commands.notes.repository
+
+import cats.effect.Async
+import doobie.implicits._
+import doobie.util.query.Query0
+import doobie.util.transactor.Transactor
+import doobie.util.update.Update0
+import ru.gaptrakhmanov.telegram.helper.commands.notes.model.User
+
+trait DoobieUserRepository[F[_]] extends UserRepository[F] {
+  protected def transactor: Transactor[F]
```

Comment:
I can understand why you added the transactor here, but I think it is not a big deal but it doesn't add too much to the current code, we could just have the transactor coming on the create method.